### PR TITLE
CXF 100801 Adding Z-side VD service token creation  and VD 2 Service Token Connection example

### PR DESCRIPTION
**Pull request by srushti-patl**
- Added Service Token Creation Example for Zside VD Token
- Added Vd 2 Service Token Example
- Updated output values for all examples

### CODE REVIEW

File: examples/virtual-device-2-service-token-connection/main.tf [hunk 1]
```diff
@@ -0,0 +1,22 @@
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+module "create_virtual_device_2_service_token_connection" {
```
Comment (thogarty) [inline examples/virtual-device-2-service-token-connection/main.tf:6]: Added in slack, but adding here as well. What's the reason that we have this example separate from the service token creation example? Wouldn't it be useful for the customer to have it all together?

**Review by thogarty — APPROVED** [review]
Approved. Thanks, @srushti-patl !

**Comment by srushti-patl** [comment]
Resolving this issue:
https://github.com/equinix/terraform-equinix-fabric/issues/139

File: examples/virtual-device-2-service-token-connection/main.tf (hunk 1 above)
Comment (srushti-patl) [reply examples/virtual-device-2-service-token-connection/main.tf:6]: We have also considered buyer seller scenario here,
Seller will create the service token and buyer will create separate connection using the same service token.

**Pull request merged**